src/main/resources/JavadocLexer.g4

```diff
@@ -73,13 +74,12 @@ import org.antlr.v4.runtime.Token;
     @Override
     public void emit(Token token) {
         super.emit(token);
-        if (token.getType() == TAG_NAME) {
+        if (token.getType() == TAG_NAME && !LexerUtility.isSelfClosing(_input)) {
```

Comment:
Probably a stupid question, but why don't we just move all this Java code to the LexerUtility since we have it already?

Reply:
> for every token we lex
> the overridden emit method

I think your focusing on the new changes, my question was more on any java code in the lexer. Another example would be `isJavadocTag`.